release-team/role-handbooks/communications/README.md

```diff
@@ -69,39 +72,84 @@ The release lead will drive the content for the release theme and logo.
 
 Tracking, facilitating, and organizing the publication of the Feature Blog series is a major deliverable of the Comms team. Feature blogs are opt-in for SIGs, and authored by enhancement developers and others close to the features. It helps to work with the release lead and use the respective SIG Slack channels to remind the SIGs about opting-in to feature blogs. 
```

Comment:
```suggestion
Tracking, facilitating, and organizing the publication of the Feature Blog series is a major deliverable of the Comms team. Feature blogs are opt-in for SIGs, and are authored by enhancement developers and others close to the features. We do, however, need to encourage owners of important enhancements to opt in to writing feature blogs. Examples of important enhancements that warrant a feature blog might include: breaking changes, features that have been in progress for a long time and are graduating, and features that are considered mandatory by the Release Lead. It helps to work closely with the Release Lead and use the respective SIG Slack channels to remind the SIGs about opting in to feature blogs and provide any necessary context to blog authors. 
```

Reply:
Enhancements that will be opted-in as a major theme may be a good candidate for a feature blog
This also gets the enhancement owner or SIG to think if their enhancement should be included in the major themes of the release

Reply:
@reylejano  could you included as suggestion in the handbook? 🙏🏾